docs/source/visualize.rst

```diff
@@ -0,0 +1,12 @@
+.. _visualize:
```

Comment:
At some point I discussed with Orel to put visualize.py inside gammasim-tools/util and it turned out it would be better to create a repository called `visualization` and put both visualize.py and util/legend_handler.py there since they are related. 
Do we want to do that now?

Reply:
Let's not mix the intention of pull requests. This is about documentation, and not about moving modules. This should be done in a separate pr.

Reply:
ok!